src/test/ui/argument-suggestions/invalid_arguments.stderr

```diff
@@ -0,0 +1,349 @@
+error[E0059]: arguments to this function are incorrect
+  --> $DIR/invalid_arguments.rs:13:11
+   |
+LL |   one_arg(1.0);
+   |           ^^^ expected i32, found {float}
+   |
+note: function defined here
+  --> $DIR/invalid_arguments.rs:5:4
+   |
+LL | fn one_arg(_a: i32) {}
+   |    ^^^^^^^ -------
+
+error[E0059]: multiple arguments to this function are incorrect
+  --> $DIR/invalid_arguments.rs:16:19
+   |
+LL |   two_arg_same(1, "");
+   |                   ^^ ^ missing argument of type i32
```

Comment:
This span seems incorrect; I think it should be pointing to the closing parenthesis.

Reply:
Yea, the biggest thing remaining is the annoying span fiddling to get things lined up correctly, I'll be working on that over the next few days.